azure/aks-full-sample/pom.xml

```diff
@@ -4,36 +4,32 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>com.yanchware.fractal</groupId>
-    <artifactId>aks.full.sample</artifactId>
-    <version>1.0-SNAPSHOT</version>
-
+    <parent>
+        <groupId>com.yanchware.fractal</groupId>
+        <artifactId>azure</artifactId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+    
+    <artifactId>azure.aks.full.sample</artifactId>
     <name>aks-full-sample</name>
     <url>https://github.com/YanchWare/fractal-samples</url>
     <description>Sample of AKS with all possible options using Fractal Cloud SDK</description>
 
-    <properties>
-        <maven.compiler.source>18</maven.compiler.source>
-        <maven.compiler.target>18</maven.compiler.target>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    </properties>
-
     <dependencies>
         <dependency>
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-lang3</artifactId>
-            <version>3.12.0</version>
```

Comment:
Nice idea to move this into the parent pom

Reply:
it should be easier to manage versions, especially SDK.